ui/src/components/delete/QasDelete.stories.js

```diff
@@ -0,0 +1,72 @@
+import QasDelete from './QasDelete.vue'
+
+export default {
+  component: QasDelete,
+  title: 'Components/Delete',
+
+  parameters: {
+    docs: {
+      description: {
+        component: 'Insert line breaks where newlines `\n` occur in the string.'
+      }
+    }
+  },
+
+  argTypes: {
+    dialog: {
+      description: 'recebe as configurações do dialog'
+    },
+    customId: {
+      description: 'CustomID receives a customized id of the item to be deleted or if not the id to be used will be from the route'
+    },
+    entity: {
+      description: 'The entity is used by vuex as a url to define which element will be deleted from the store. has as parameter the ID.'
```

Comment:
`Entity of vuex` ficaria mais generico não acha? lembrando que a entidade pode ou não ser usada como URL.

Reply:
concordo, mais não é bom explicar o que faz, ou colocar um link?